### ISSUE TITLE
UX suggestion: Add NEXT JOB navigation link to Job view

### ISSUE THREAD

**Issue by mclamb** [digitalrebar/provision#142]
It would be nice to have a NEXT JOB link similar to the PREVIOUS JOB link in the Job view. I find that when a job fails I will often fix content, re-upload, then click on the green "Set Runnable" icon to-rerun this job. I then have to navigate back to the Jobs list, refresh, then go into the new job view. Instead of this flow it would be nice to just click "Set Runnable" and then have a NEXT JOB link pop up that can take me to the now-running job view.

**Comment by sygibson** [comment]
@mclamb  - thank you for the suggestion ... there is "next job" link as long as you are not on the current running job.  The problem with "next job" during current running job ... we don't have the Job ID to link to the job log, as it hasn't been generated yet.  A JID will only be generated when the job is created.

**Comment by mclamb** [comment]
Yes I was just noticing that. Changing the title to more accurately reflect this. Thanks!

**Comment by mclamb** [comment]
To better capture the discussion from yesterday on Slack with @zehicle, the primary issue / feature request here is for the UX to figure out the forward path when clicking on a Job. The act of clicking on a Job makes that Job the last element in a linked list of Jobs. You can navigate backwards for the entire history of that machine and you can navigate forwards from any of those jobs all the way back to this clicked-on Job. However you cannot navigate forwards in history once a Job has been clicked, even if those Jobs are historical as well.

A nice addition to this capability would be to dynamically add a Next Job advance for newly created Jobs if the current view is on the most recent Job.
